setup.py

```diff
@@ -7,6 +7,7 @@
 
 dependencies = [
     "clvm>=0.9.2",
+    "clvm_rs>=0.0.0"
```

Comment:
I think it's best to not depend upon `clvm_rs` at this point. I'm still hoping to add dialect support to it.